Index: domoticz-bc7445ff9dd3bae50c478c7bf75ed2fb7594b388/hardware/Comm5Serial.cpp
===================================================================
--- domoticz-bc7445ff9dd3bae50c478c7bf75ed2fb7594b388.orig/hardware/Comm5Serial.cpp
+++ domoticz-bc7445ff9dd3bae50c478c7bf75ed2fb7594b388/hardware/Comm5Serial.cpp
@@ -5,7 +5,7 @@
 #include "../main/Logger.h"
 #include "../main/RFXtrx.h"
 
-using namespace boost::placeholders;
+using namespace std::placeholders;
 
 /*
 	This driver allows Domoticz to control any I/O module from the MA-4xxx Family
@@ -94,7 +94,7 @@ bool Comm5Serial::StartHardware()
 		return false;
 	}
 	m_bIsStarted=true;
-	setReadCallback(boost::bind(&Comm5Serial::readCallBack, this, _1, _2));
+	setReadCallback(std::bind(&Comm5Serial::readCallBack, this, _1, _2));
 
 	sOnConnected(this);
 	return true;
Index: domoticz-bc7445ff9dd3bae50c478c7bf75ed2fb7594b388/hardware/DenkoviUSBDevices.cpp
===================================================================
--- domoticz-bc7445ff9dd3bae50c478c7bf75ed2fb7594b388.orig/hardware/DenkoviUSBDevices.cpp
+++ domoticz-bc7445ff9dd3bae50c478c7bf75ed2fb7594b388/hardware/DenkoviUSBDevices.cpp
@@ -5,12 +5,12 @@
 #include "../main/localtime_r.h"
 #include "../main/mainworker.h"
 
-using namespace boost::placeholders;
-
 #define MAX_POLL_INTERVAL 3600*1000
 
 #define DAE_IO_TYPE_RELAY		2
 
+using namespace std::placeholders;
+
 CDenkoviUSBDevices::CDenkoviUSBDevices(const int ID, const std::string& comPort, const int model) :
 	m_szSerialPort(comPort)
 {
@@ -68,7 +68,7 @@ bool CDenkoviUSBDevices::StartHardware()
 	m_thread = std::make_shared<std::thread>(&CDenkoviUSBDevices::Do_Work, this);
 
 	m_bIsStarted = true;
-	setReadCallback(boost::bind(&CDenkoviUSBDevices::readCallBack, this, _1, _2));
+	setReadCallback(std::bind(&CDenkoviUSBDevices::readCallBack, this, _1, _2));
 	
 	sOnConnected(this);
 	return true;
Index: domoticz-bc7445ff9dd3bae50c478c7bf75ed2fb7594b388/hardware/EvohomeRadio.cpp
===================================================================
--- domoticz-bc7445ff9dd3bae50c478c7bf75ed2fb7594b388.orig/hardware/EvohomeRadio.cpp
+++ domoticz-bc7445ff9dd3bae50c478c7bf75ed2fb7594b388/hardware/EvohomeRadio.cpp
@@ -25,10 +25,10 @@
 #include <json/json.h>
 #include <boost/bind/bind.hpp>
 
-using namespace boost::placeholders;
-
 extern std::string szUserDataFolder;
 
+using namespace std::placeholders;
+
 enum evoCommands
 {
 	cmdSysInfo = 0x10e0,
@@ -88,28 +88,28 @@ CEvohomeRadio::CEvohomeRadio(const int I
 		s_strid >> std::hex >> m_UControllerID;
 	}
 
-	RegisterDecoder(cmdZoneTemp, boost::bind(&CEvohomeRadio::DecodeZoneTemp, this, _1));
-	RegisterDecoder(cmdSetPoint, boost::bind(&CEvohomeRadio::DecodeSetpoint, this, _1));
-	RegisterDecoder(cmdSetpointOverride, boost::bind(&CEvohomeRadio::DecodeSetpointOverride, this, _1));
-	RegisterDecoder(cmdDHWState, boost::bind(&CEvohomeRadio::DecodeDHWState, this, _1));
-	RegisterDecoder(cmdDHWTemp, boost::bind(&CEvohomeRadio::DecodeDHWTemp, this, _1));
-        RegisterDecoder(cmdDHWSettings, boost::bind(&CEvohomeRadio::DecodeDHWSettings, this, _1));
-	RegisterDecoder(cmdControllerMode, boost::bind(&CEvohomeRadio::DecodeControllerMode, this, _1));
-	RegisterDecoder(cmdSysInfo, boost::bind(&CEvohomeRadio::DecodeSysInfo, this, _1));
-	RegisterDecoder(cmdZoneName, boost::bind(&CEvohomeRadio::DecodeZoneName, this, _1));
-	RegisterDecoder(cmdZoneHeatDemand, boost::bind(&CEvohomeRadio::DecodeHeatDemand, this, _1));
-	RegisterDecoder(cmdOpenThermBridge, boost::bind(&CEvohomeRadio::DecodeOpenThermBridge, this, _1));
-	RegisterDecoder(cmdOpenThermSetpoint, boost::bind(&CEvohomeRadio::DecodeOpenThermSetpoint, this, _1));
- 	RegisterDecoder(cmdZoneInfo, boost::bind(&CEvohomeRadio::DecodeZoneInfo, this, _1));
-	RegisterDecoder(cmdControllerHeatDemand, boost::bind(&CEvohomeRadio::DecodeHeatDemand, this, _1));
-	RegisterDecoder(cmdBinding, boost::bind(&CEvohomeRadio::DecodeBinding, this, _1));
-	RegisterDecoder(cmdActuatorState, boost::bind(&CEvohomeRadio::DecodeActuatorState, this, _1));
-	RegisterDecoder(cmdActuatorCheck, boost::bind(&CEvohomeRadio::DecodeActuatorCheck, this, _1));
-	RegisterDecoder(cmdZoneWindow, boost::bind(&CEvohomeRadio::DecodeZoneWindow, this, _1));
-	RegisterDecoder(cmdExternalSensor, boost::bind(&CEvohomeRadio::DecodeExternalSensor, this, _1));
-	RegisterDecoder(cmdDeviceInfo, boost::bind(&CEvohomeRadio::DecodeDeviceInfo, this, _1));
-	RegisterDecoder(cmdBatteryInfo, boost::bind(&CEvohomeRadio::DecodeBatteryInfo, this, _1));
-	RegisterDecoder(cmdSync, boost::bind(&CEvohomeRadio::DecodeSync, this, _1));
+	RegisterDecoder(cmdZoneTemp, std::bind(&CEvohomeRadio::DecodeZoneTemp, this, _1));
+	RegisterDecoder(cmdSetPoint, std::bind(&CEvohomeRadio::DecodeSetpoint, this, _1));
+	RegisterDecoder(cmdSetpointOverride, std::bind(&CEvohomeRadio::DecodeSetpointOverride, this, _1));
+	RegisterDecoder(cmdDHWState, std::bind(&CEvohomeRadio::DecodeDHWState, this, _1));
+	RegisterDecoder(cmdDHWTemp, std::bind(&CEvohomeRadio::DecodeDHWTemp, this, _1));
+        RegisterDecoder(cmdDHWSettings, std::bind(&CEvohomeRadio::DecodeDHWSettings, this, _1));
+	RegisterDecoder(cmdControllerMode, std::bind(&CEvohomeRadio::DecodeControllerMode, this, _1));
+	RegisterDecoder(cmdSysInfo, std::bind(&CEvohomeRadio::DecodeSysInfo, this, _1));
+	RegisterDecoder(cmdZoneName, std::bind(&CEvohomeRadio::DecodeZoneName, this, _1));
+	RegisterDecoder(cmdZoneHeatDemand, std::bind(&CEvohomeRadio::DecodeHeatDemand, this, _1));
+	RegisterDecoder(cmdOpenThermBridge, std::bind(&CEvohomeRadio::DecodeOpenThermBridge, this, _1));
+	RegisterDecoder(cmdOpenThermSetpoint, std::bind(&CEvohomeRadio::DecodeOpenThermSetpoint, this, _1));
+ 	RegisterDecoder(cmdZoneInfo, std::bind(&CEvohomeRadio::DecodeZoneInfo, this, _1));
+	RegisterDecoder(cmdControllerHeatDemand, std::bind(&CEvohomeRadio::DecodeHeatDemand, this, _1));
+	RegisterDecoder(cmdBinding, std::bind(&CEvohomeRadio::DecodeBinding, this, _1));
+	RegisterDecoder(cmdActuatorState, std::bind(&CEvohomeRadio::DecodeActuatorState, this, _1));
+	RegisterDecoder(cmdActuatorCheck, std::bind(&CEvohomeRadio::DecodeActuatorCheck, this, _1));
+	RegisterDecoder(cmdZoneWindow, std::bind(&CEvohomeRadio::DecodeZoneWindow, this, _1));
+	RegisterDecoder(cmdExternalSensor, std::bind(&CEvohomeRadio::DecodeExternalSensor, this, _1));
+	RegisterDecoder(cmdDeviceInfo, std::bind(&CEvohomeRadio::DecodeDeviceInfo, this, _1));
+	RegisterDecoder(cmdBatteryInfo, std::bind(&CEvohomeRadio::DecodeBatteryInfo, this, _1));
+	RegisterDecoder(cmdSync, std::bind(&CEvohomeRadio::DecodeSync, this, _1));
 }
 
 CEvohomeRadio::~CEvohomeRadio()
Index: domoticz-bc7445ff9dd3bae50c478c7bf75ed2fb7594b388/hardware/EvohomeSerial.cpp
===================================================================
--- domoticz-bc7445ff9dd3bae50c478c7bf75ed2fb7594b388.orig/hardware/EvohomeSerial.cpp
+++ domoticz-bc7445ff9dd3bae50c478c7bf75ed2fb7594b388/hardware/EvohomeSerial.cpp
@@ -7,7 +7,7 @@
 #include "../main/localtime_r.h"
 #include <boost/exception/diagnostic_information.hpp>
 
-using namespace boost::placeholders;
+using namespace std::placeholders;
 
 CEvohomeSerial::CEvohomeSerial(const int ID, const std::string &szSerialPort, const int baudrate, const std::string &UserContID) :
 CEvohomeRadio(ID, UserContID)
@@ -66,7 +66,7 @@ bool CEvohomeSerial::OpenSerialDevice()
 	}
 	m_nBufPtr=0;
 	m_bIsStarted=true;
-	setReadCallback(boost::bind(&CEvohomeSerial::ReadCallback, this, _1, _2));
+	setReadCallback(std::bind(&CEvohomeSerial::ReadCallback, this, _1, _2));
 	sOnConnected(this);
 	return true;
 }
Index: domoticz-bc7445ff9dd3bae50c478c7bf75ed2fb7594b388/hardware/MQTT.cpp
===================================================================
--- domoticz-bc7445ff9dd3bae50c478c7bf75ed2fb7594b388.orig/hardware/MQTT.cpp
+++ domoticz-bc7445ff9dd3bae50c478c7bf75ed2fb7594b388/hardware/MQTT.cpp
@@ -11,8 +11,6 @@
 #define __STDC_FORMAT_MACROS
 #include <inttypes.h>
 
-using namespace boost::placeholders;
-
 #define RETRY_DELAY 30
 
 #define CLIENTID	"Domoticz"
@@ -20,6 +18,8 @@ using namespace boost::placeholders;
 #define TOPIC_IN	"domoticz/in"
 #define QOS         1
 
+using namespace std::placeholders;
+
 const char* szTLSVersions[3] =
 {
 	"tlsv1",
@@ -153,8 +153,8 @@ void MQTT::on_connect(int rc)
 			_log.Log(LOG_STATUS, "MQTT: connected to: %s:%d", m_szIPAddress.c_str(), m_usIPPort);
 			m_IsConnected = true;
 			sOnConnected(this);
-			m_sDeviceReceivedConnection = m_mainworker.sOnDeviceReceived.connect(boost::bind(&MQTT::SendDeviceInfo, this, _1, _2, _3, _4));
-			m_sSwitchSceneConnection = m_mainworker.sOnSwitchScene.connect(boost::bind(&MQTT::SendSceneInfo, this, _1, _2));
+			m_sDeviceReceivedConnection = m_mainworker.sOnDeviceReceived.connect(std::bind(&MQTT::SendDeviceInfo, this, _1, _2, _3, _4));
+			m_sSwitchSceneConnection = m_mainworker.sOnSwitchScene.connect(std::bind(&MQTT::SendSceneInfo, this, _1, _2));
 		}
 		subscribe(nullptr, m_TopicIn.c_str());
 	}
Index: domoticz-bc7445ff9dd3bae50c478c7bf75ed2fb7594b388/hardware/Pinger.cpp
===================================================================
--- domoticz-bc7445ff9dd3bae50c478c7bf75ed2fb7594b388.orig/hardware/Pinger.cpp
+++ domoticz-bc7445ff9dd3bae50c478c7bf75ed2fb7594b388/hardware/Pinger.cpp
@@ -19,14 +19,14 @@
 
 #include <iostream>
 
-using namespace boost::placeholders;
-
 #if BOOST_VERSION >= 107000
 #define GET_IO_SERVICE(s) ((boost::asio::io_context&)(s).get_executor().context())
 #else
 #define GET_IO_SERVICE(s) ((s).get_io_service())
 #endif
 
+using namespace std::placeholders;
+
 class pinger
 	: private domoticz::noncopyable
 {
@@ -93,7 +93,7 @@ private:
 				}
 				else
 				{
-					timer_.async_wait(boost::bind(&pinger::start_send, this));
+					timer_.async_wait(std::bind(&pinger::start_send, this));
 				}
 			}
 		}
@@ -106,7 +106,7 @@ private:
 
 		// Wait for a reply. We prepare the buffer to receive up to 64KB.
 		socket_.async_receive(reply_buffer_.prepare(65536),
-			boost::bind(&pinger::handle_receive, this, _2));
+			std::bind(&pinger::handle_receive, this, _2));
 	}
 
 	void handle_receive(std::size_t length)
@@ -402,7 +402,7 @@ void CPinger::DoPingHosts()
 		if (m_iThreadsRunning < 1000)
 		{
 			//m_iThreadsRunning++;
-			boost::thread t(boost::bind(&CPinger::Do_Ping_Worker, this, node));
+			boost::thread t(std::bind(&CPinger::Do_Ping_Worker, this, node));
 			SetThreadName(t.native_handle(), "PingerWorker");
 			t.join();
 		}
Index: domoticz-bc7445ff9dd3bae50c478c7bf75ed2fb7594b388/hardware/RAVEn.cpp
===================================================================
--- domoticz-bc7445ff9dd3bae50c478c7bf75ed2fb7594b388.orig/hardware/RAVEn.cpp
+++ domoticz-bc7445ff9dd3bae50c478c7bf75ed2fb7594b388/hardware/RAVEn.cpp
@@ -8,11 +8,11 @@
 #include "../tinyxpath/tinyxml.h"
 #include "hardwaretypes.h"
 
-using namespace boost::placeholders;
-
 //Rainforest RAVEn USB ZigBee Smart Meter Adapter
 //https://rainforestautomation.com/rfa-z106-raven/
 
+using namespace std::placeholders;
+
 RAVEn::RAVEn(const int ID, const std::string& devname)
 : device_(devname), m_wptr(m_buffer), m_currUsage(0), m_totalUsage(0)
 {
@@ -44,7 +44,7 @@ bool RAVEn::StartHardware()
         _log.Log(LOG_ERROR, "RAVEn: Error opening serial port!!!");
         return false;
     }
-    setReadCallback(boost::bind(&RAVEn::readCallback, this, _1, _2));
+    setReadCallback(std::bind(&RAVEn::readCallback, this, _1, _2));
     m_bIsStarted = true;
     sOnConnected(this);
 
Index: domoticz-bc7445ff9dd3bae50c478c7bf75ed2fb7594b388/hardware/RFLinkSerial.cpp
===================================================================
--- domoticz-bc7445ff9dd3bae50c478c7bf75ed2fb7594b388.orig/hardware/RFLinkSerial.cpp
+++ domoticz-bc7445ff9dd3bae50c478c7bf75ed2fb7594b388/hardware/RFLinkSerial.cpp
@@ -5,7 +5,7 @@
 #include "../main/localtime_r.h"
 #include <boost/exception/diagnostic_information.hpp>
 
-using namespace boost::placeholders;
+using namespace std::placeholders;
 
 CRFLinkSerial::CRFLinkSerial(const int ID, const std::string& devname) :
 m_szSerialPort(devname)
@@ -179,7 +179,7 @@ bool CRFLinkSerial::OpenSerialDevice()
 	m_rfbufferpos = 0;
 	m_LastReceivedTime = mytime(nullptr);
 
-	setReadCallback(boost::bind(&CRFLinkSerial::readCallback, this, _1, _2));
+	setReadCallback(std::bind(&CRFLinkSerial::readCallback, this, _1, _2));
 	sOnConnected(this);
 
 	return true;
Index: domoticz-bc7445ff9dd3bae50c478c7bf75ed2fb7594b388/hardware/TCPProxy/tcpproxy_server.cpp
===================================================================
--- domoticz-bc7445ff9dd3bae50c478c7bf75ed2fb7594b388.orig/hardware/TCPProxy/tcpproxy_server.cpp
+++ domoticz-bc7445ff9dd3bae50c478c7bf75ed2fb7594b388/hardware/TCPProxy/tcpproxy_server.cpp
@@ -15,14 +15,14 @@
 #include "stdafx.h"
 #include "tcpproxy_server.h"
 
-using namespace boost::placeholders;
-
 #if BOOST_VERSION >= 107000
 #define GET_IO_SERVICE(s) ((boost::asio::io_context&)(s).get_executor().context())
 #else
 #define GET_IO_SERVICE(s) ((s).get_io_service())
 #endif
 
+using namespace std::placeholders;
+
 namespace tcp_proxy
 {
 	bridge::bridge(boost::asio::io_service& ios)
@@ -180,8 +180,8 @@ namespace tcp_proxy
 		try
 		{
 			session_ = std::make_shared<bridge>(io_service_);
-			session_->sDownstreamData.connect( boost::bind( &acceptor::OnDownstreamData, this, _1, _2 ) );
-			session_->sUpstreamData.connect( boost::bind( &acceptor::OnUpstreamData, this, _1, _2 ) );
+			session_->sDownstreamData.connect( std::bind( &acceptor::OnDownstreamData, this, _1, _2 ) );
+			session_->sUpstreamData.connect( std::bind( &acceptor::OnUpstreamData, this, _1, _2 ) );
 
 			acceptor_.async_accept(session_->downstream_socket(),
 				boost::bind(&acceptor::handle_accept,
Index: domoticz-bc7445ff9dd3bae50c478c7bf75ed2fb7594b388/hardware/ZiBlueSerial.cpp
===================================================================
--- domoticz-bc7445ff9dd3bae50c478c7bf75ed2fb7594b388.orig/hardware/ZiBlueSerial.cpp
+++ domoticz-bc7445ff9dd3bae50c478c7bf75ed2fb7594b388/hardware/ZiBlueSerial.cpp
@@ -6,10 +6,10 @@
 
 #include <boost/exception/diagnostic_information.hpp>
 
-using namespace boost::placeholders;
-
 #define ZiBlue_RETRY_DELAY 30
 
+using namespace std::placeholders;
+
 CZiBlueSerial::CZiBlueSerial(const int ID, const std::string& devname) :
 m_szSerialPort(devname)
 {
@@ -133,7 +133,7 @@ bool CZiBlueSerial::OpenSerialDevice()
 	m_rfbufferpos = 0;
 	m_LastReceivedTime = mytime(nullptr);
 
-	setReadCallback(boost::bind(&CZiBlueSerial::readCallback, this, _1, _2));
+	setReadCallback(std::bind(&CZiBlueSerial::readCallback, this, _1, _2));
 
 	sOnConnected(this);
 
Index: domoticz-bc7445ff9dd3bae50c478c7bf75ed2fb7594b388/hardware/plugins/PluginTransports.cpp
===================================================================
--- domoticz-bc7445ff9dd3bae50c478c7bf75ed2fb7594b388.orig/hardware/plugins/PluginTransports.cpp
+++ domoticz-bc7445ff9dd3bae50c478c7bf75ed2fb7594b388/hardware/plugins/PluginTransports.cpp
@@ -13,7 +13,7 @@
 #include "icmp_header.hpp"
 #include "ipv4_header.hpp"
 
-using namespace boost::placeholders;
+using namespace std::placeholders;
 
 namespace Plugins {
 
@@ -972,7 +972,7 @@ namespace Plugins {
 				if (m_bConnected)
 				{
 					pPlugin->MessagePlugin(new onConnectCallback(pPlugin, m_pConnection, 0, "SerialPort " + m_Port + " opened successfully."));
-					setReadCallback(boost::bind(&CPluginTransportSerial::handleRead, this, _1, _2));
+					setReadCallback(std::bind(&CPluginTransportSerial::handleRead, this, _1, _2));
 				}
 				else
 				{
Index: domoticz-bc7445ff9dd3bae50c478c7bf75ed2fb7594b388/push/FibaroPush.cpp
===================================================================
--- domoticz-bc7445ff9dd3bae50c478c7bf75ed2fb7594b388.orig/push/FibaroPush.cpp
+++ domoticz-bc7445ff9dd3bae50c478c7bf75ed2fb7594b388/push/FibaroPush.cpp
@@ -14,7 +14,7 @@
 #define __STDC_FORMAT_MACROS
 #include <inttypes.h>
 
-using namespace boost::placeholders;
+using namespace std::placeholders;
 
 CFibaroPush::CFibaroPush()
 {
@@ -25,7 +25,7 @@ CFibaroPush::CFibaroPush()
 void CFibaroPush::Start()
 {
 	UpdateActive();
-	m_sConnection = m_mainworker.sOnDeviceReceived.connect(boost::bind(&CFibaroPush::OnDeviceReceived, this, _1, _2, _3, _4));
+	m_sConnection = m_mainworker.sOnDeviceReceived.connect(std::bind(&CFibaroPush::OnDeviceReceived, this, _1, _2, _3, _4));
 }
 
 void CFibaroPush::Stop()
Index: domoticz-bc7445ff9dd3bae50c478c7bf75ed2fb7594b388/push/GooglePubSubPush.cpp
===================================================================
--- domoticz-bc7445ff9dd3bae50c478c7bf75ed2fb7594b388.orig/push/GooglePubSubPush.cpp
+++ domoticz-bc7445ff9dd3bae50c478c7bf75ed2fb7594b388/push/GooglePubSubPush.cpp
@@ -21,7 +21,7 @@ extern "C" {
 #include <boost/python.hpp>
 #endif
 
-using namespace boost::placeholders;
+using namespace std::placeholders;
 
 extern std::string szUserDataFolder;
 
@@ -41,7 +41,7 @@ CGooglePubSubPush::CGooglePubSubPush()
 void CGooglePubSubPush::Start()
 {
 	UpdateActive();
-	m_sConnection = m_mainworker.sOnDeviceReceived.connect(boost::bind(&CGooglePubSubPush::OnDeviceReceived, this, _1, _2, _3, _4));
+	m_sConnection = m_mainworker.sOnDeviceReceived.connect(std::bind(&CGooglePubSubPush::OnDeviceReceived, this, _1, _2, _3, _4));
 }
 
 void CGooglePubSubPush::Stop()
Index: domoticz-bc7445ff9dd3bae50c478c7bf75ed2fb7594b388/push/HttpPush.cpp
===================================================================
--- domoticz-bc7445ff9dd3bae50c478c7bf75ed2fb7594b388.orig/push/HttpPush.cpp
+++ domoticz-bc7445ff9dd3bae50c478c7bf75ed2fb7594b388/push/HttpPush.cpp
@@ -15,7 +15,7 @@
 #define __STDC_FORMAT_MACROS
 #include <inttypes.h>
 
-using namespace boost::placeholders;
+using namespace std::placeholders;
 
 CHttpPush::CHttpPush()
 {
@@ -26,7 +26,7 @@ CHttpPush::CHttpPush()
 void CHttpPush::Start()
 {
 	UpdateActive();
-	m_sConnection = m_mainworker.sOnDeviceReceived.connect(boost::bind(&CHttpPush::OnDeviceReceived, this, _1, _2, _3, _4));
+	m_sConnection = m_mainworker.sOnDeviceReceived.connect(std::bind(&CHttpPush::OnDeviceReceived, this, _1, _2, _3, _4));
 }
 
 void CHttpPush::Stop()
Index: domoticz-bc7445ff9dd3bae50c478c7bf75ed2fb7594b388/push/InfluxPush.cpp
===================================================================
--- domoticz-bc7445ff9dd3bae50c478c7bf75ed2fb7594b388.orig/push/InfluxPush.cpp
+++ domoticz-bc7445ff9dd3bae50c478c7bf75ed2fb7594b388/push/InfluxPush.cpp
@@ -15,7 +15,7 @@
 #define __STDC_FORMAT_MACROS
 #include <inttypes.h>
 
-using namespace boost::placeholders;
+using namespace std::placeholders;
 
 CInfluxPush::CInfluxPush()
 {
@@ -34,7 +34,7 @@ bool CInfluxPush::Start()
 	m_thread = std::make_shared<std::thread>(&CInfluxPush::Do_Work, this);
 	SetThreadName(m_thread->native_handle(), "InfluxPush");
 
-	m_sConnection = m_mainworker.sOnDeviceReceived.connect(boost::bind(&CInfluxPush::OnDeviceReceived, this, _1, _2, _3, _4));
+	m_sConnection = m_mainworker.sOnDeviceReceived.connect(std::bind(&CInfluxPush::OnDeviceReceived, this, _1, _2, _3, _4));
 
 	return (m_thread != nullptr);
 }
Index: domoticz-bc7445ff9dd3bae50c478c7bf75ed2fb7594b388/push/WebsocketPush.cpp
===================================================================
--- domoticz-bc7445ff9dd3bae50c478c7bf75ed2fb7594b388.orig/push/WebsocketPush.cpp
+++ domoticz-bc7445ff9dd3bae50c478c7bf75ed2fb7594b388/push/WebsocketPush.cpp
@@ -3,7 +3,7 @@
 #include "../webserver/WebsocketHandler.h"
 #include "../main/mainworker.h"
 
-using namespace boost::placeholders;
+using namespace std::placeholders;
 
 extern boost::signals2::signal<void(const std::string &Subject, const std::string &Text, const std::string &ExtraData, const int Priority, const std::string & Sound, const bool bFromNotification)> sOnNotificationReceived;
 
@@ -22,10 +22,10 @@ void CWebSocketPush::Start()
 	if (isStarted) {
 		return;
 	}
-	m_sConnection = m_mainworker.sOnDeviceReceived.connect(boost::bind(&CWebSocketPush::OnDeviceReceived, this, _1, _2, _3, _4));
-	m_sDeviceUpdate = m_mainworker.sOnDeviceUpdate.connect(boost::bind(&CWebSocketPush::OnDeviceUpdate, this, _1, _2));
-	m_sNotification = sOnNotificationReceived.connect(boost::bind(&CWebSocketPush::OnNotificationReceived, this, _1, _2, _3, _4, _5, _6));
-	m_sSceneChanged = m_mainworker.sOnSwitchScene.connect(boost::bind(&CWebSocketPush::OnSceneChange, this, _1, _2));
+	m_sConnection = m_mainworker.sOnDeviceReceived.connect(std::bind(&CWebSocketPush::OnDeviceReceived, this, _1, _2, _3, _4));
+	m_sDeviceUpdate = m_mainworker.sOnDeviceUpdate.connect(std::bind(&CWebSocketPush::OnDeviceUpdate, this, _1, _2));
+	m_sNotification = sOnNotificationReceived.connect(std::bind(&CWebSocketPush::OnNotificationReceived, this, _1, _2, _3, _4, _5, _6));
+	m_sSceneChanged = m_mainworker.sOnSwitchScene.connect(std::bind(&CWebSocketPush::OnSceneChange, this, _1, _2));
 	isStarted = true;
 }
 
Index: domoticz-bc7445ff9dd3bae50c478c7bf75ed2fb7594b388/webserver/proxyclient.cpp
===================================================================
--- domoticz-bc7445ff9dd3bae50c478c7bf75ed2fb7594b388.orig/webserver/proxyclient.cpp
+++ domoticz-bc7445ff9dd3bae50c478c7bf75ed2fb7594b388/webserver/proxyclient.cpp
@@ -8,8 +8,6 @@
 #include "../tcpserver/TCPServer.h"
 #include "sha1.hpp"
 
-using namespace boost::placeholders;
-
 // RK: some defines to make mydomoticz also work when openssl not compiled in
 #ifdef WWW_ENABLE_SSL
 #define PROXY_PORT 443
@@ -21,6 +19,8 @@ using namespace boost::placeholders;
 
 extern std::string szAppVersion;
 
+using namespace std::placeholders;
+
 #define PROXY_TIMEOUT 60
 #define PONG "PONG"
 
@@ -350,7 +350,7 @@ namespace http {
 			// todo: make a map of websocket connections. There can be more than one.
 			// open new virtual websocket connection
 			// todo: different request_url's can have different websocket handlers
-			websocket_handlers[pdu->m_requestid] = new CWebsocketHandler(m_pWebEm, boost::bind(&CProxyClient::WS_Write, this, pdu->m_requestid, _1));
+			websocket_handlers[pdu->m_requestid] = new CWebsocketHandler(m_pWebEm, std::bind(&CProxyClient::WS_Write, this, pdu->m_requestid, _1));
 			websocket_handlers[pdu->m_requestid]->Start();
 		}
 
