Index: domoticz-805fee10fd609a02182eaa01e99bcb4b1b560426/hardware/MySensorsBase.cpp
===================================================================
--- domoticz-805fee10fd609a02182eaa01e99bcb4b1b560426.orig/hardware/MySensorsBase.cpp
+++ domoticz-805fee10fd609a02182eaa01e99bcb4b1b560426/hardware/MySensorsBase.cpp
@@ -1278,7 +1278,7 @@ bool MySensorsBase::SendNodeSetCommandIm
 	m_AckChildID = ChildID;
 	m_AckSetType = SubType;
 	int repeat = 0;
-	int repeats = 2;
+	int repeats = 5;
 
 	//Resend failed command
 	while ((!m_bAckReceived) && (repeat < repeats))
